Tests/Unit/TestHelpers.swift

```diff
@@ -102,21 +102,20 @@ class KeyStoreMock: KeyStore {
 }
 
 class APIClientMock: APIClient {
-    var dictionary: [String: Any]?
+    var data: Data?
     var headers: [String: String]?
     var error: Error?
     var request: URLRequest?
 
     override func send<T>(request: URLRequest, parser: T.Type, completionHandler: @escaping (Result<Response, Error>) -> Void) where T: Parsable {
         self.request = request
 
-        guard let dictionary = dictionary else {
+        guard let data = data else {
```

Comment:
you can also add let url = request.url in this guard, so you don't need to force unwrap it later here:
if let httpResponse = HTTPURLResponse(url: request.url!, ...)

Reply:
Thanks, good suggestion.